iceoryx_posh/test/moduletests/test_capro_message.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019-2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/capro/capro_message.hpp"
+#include "iceoryx_posh/internal/roudi/introspection/port_introspection.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using namespace iox::capro;
+
+class CaproMessage_test : public Test
+{
+  public:
+    void SetUp(){};
+    void TearDown(){};
+};
+
+TEST_F(CaproMessage_test, CtorNoParams)
```

Comment:
Since the default constructor of `CaproMessage` is initializing `m_requestPort` to `nullptr` which is already done directly in the header I would suggest to remove the default constructor implementation and let the compiler generate it like:
```
CaproMessage() =default;
```

With that done we could also remove the `CtorNoParams` test since the compiler is generating everything for us and we do not test compiler generated code.

Reply:
done